SCClassLibrary/JITLib/ProxySpace/BusPlug.sc

```diff
@@ -256,6 +256,7 @@ BusPlug : AbstractFunction {
 			^this
 		};
 		if(bus.rate == \control) { "Can't monitor a control rate bus.".warn; monitor.stop; ^this };
+		group = group ?? {this.homeServer.defaultGroup};
```

Comment:
Thanks for the clarification, james. So my explanation was slightly off but the effect is the same :) (?) 
I'll keep in mind `{/**/}.def.dumpByteCode;`:

```
{#[nil, 1].choose ?? {s.defaultGroup}}.def.dumpByteCodes

>>>
BYTECODES: (13)
  0   40       PushLiteral instance of Array (0x11fbdf828, size=2, set=2)
  1   C1 24    SendSpecialMsg 'choose'
  3   8F 17 00 05 ControlOpcode 5  (11)
  7   01 14    PushInstVarX 's'
  9   B0       TailCallReturnFromFunction
 10   A1 00    SendMsg 'defaultGroup'
 12   F2       BlockReturn
```

```
{#[nil, 1].choose ? s.defaultGroup}.def.dumpByteCodes

>>>
BYTECODES: (10)
  0   40       PushLiteral instance of Array (0x11fbdf278, size=2, set=2)
  1   C1 24    SendSpecialMsg 'choose'
  3   01 14    PushInstVarX 's'
  5   A1 00    SendMsg 'defaultGroup'
  7   8F 16    ControlOpcode
  9   F2       BlockReturn
```

But I have to say that I cannot read this... what's the `ControlOpCode 5 (11)` doing? That's the jump you're talking about?